configure.ac

```diff
@@ -3,7 +3,8 @@ AC_PREREQ([2.63])
 AC_INIT(udftools, 2.0, , , [https://github.com/pali/udftools/])
 AC_CONFIG_AUX_DIR(config)
 AM_CONFIG_HEADER(include/config.h:include/config.in)
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([subdir-objects])
+AC_CONFIG_MACRO_DIRS([m4])
```

Comment:
Also this change breaks compilation, see Travis.

Reply:
It is above gitthub "comment" area. Now I reopened pull request, so travis started testing it again and you would see result there.